s/types.ts

```diff
@@ -35,19 +35,23 @@ export namespace Nub {
 	}
 }
 
+export interface ActionBinds {
+	[action: string]: string[]
+}
+
 export interface Bindings {
 
 	/** comments */
 	"👼": string[]
 
 	/** mouse action bindings */
-	"🖱️": {[action: string]: string[]}
+	"🖱": ActionBinds
 
 	/** vector2 action bindings */
-	"🕹️": {[action: string]: string[]}
+	"🕹️": ActionBinds
 
 	/** key action bindings */
-	"*️⃣": {[action: string]: string[]}
+	"*️⃣": ActionBinds
 }
```

Comment:
i've taken to calling the emojis "channels", which have "binds"..

let's set some terminology for the action binds syntax:

```
🕹️ look :: alpha bravo
```

`🕹️` — channel  
`look` — action  
`alpha` — item  
`bravo` — item  

"bind" — a record of the relationship between an "action" and any number of "items"

:thinking: note: i did use the term `name` a lot, where now i would prefer `item`.. we'll probably refactor it to `item` sooner or later